content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -99,10 +99,12 @@ limitation and compatibility rules will change.
 
 #### Pause Image
 
-Microsoft maintains a Windows pause infrastructure container at
-`mcr.microsoft.com/oss/kubernetes/pause:3.4.1`.
-Kubernetes maintains a multi-architecture image `k8s.gcr.io/pause:3.5` that
-supports Linux as well as Windows.
+Kubernetes maintains a multi-architecture image that includes support for Windows at `k8s.gcr.io/pause:3.5`.
+Source for the pause image can be found  [here](https://github.com/kubernetes/kubernetes/tree/master/build/pause).
```

Comment:
I am not sure the best way to phrase this but we should avoid linking on the word `here` to better support screen readers.

Reply:
I had the same thought. Let me try and re-word this area.